stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -56,8 +60,24 @@ public long getOperationCount() {
         return operationCount.get();
     }
 
+    /**
```

Comment:
We need to document what happens when you get a failure during doRun. Because then the afterRun is not called.